src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -190,4 +205,14 @@ private static void TranslatePatchNames()
         _ = TranslationServer.Translate("PATCH_ICE_SHELF");
         _ = TranslationServer.Translate("PATCH_PANGONIAN_SEAFLOOR");
     }
+
+    private static LocalizedStringBuilder GetPatchLocalizedName(string name, string localizedBiome)
```

Comment:
localizedBiome is not actually localized here, it should be named something like `biomeTranslationKey`